README.md

```diff
@@ -223,3 +225,81 @@ Format | Data
   etl  | Trace file (Windows only)
```

Comment:
Wrong comment location, but can't add it higher 😞 Anyway, the point: missing a closing `**` at the end of line 216